helm_tests/other/test_redis.py

```diff
@@ -357,6 +357,16 @@ def test_should_use_empty_dir_on_persistence_disabled(self, redis_values, expect
         )
         assert {"name": "redis-db", **expected} in jmespath.search("spec.template.spec.volumes", docs[0])
 
+    def test_priority_class_name(self):
+        docs = render_chart(
+            values={"redis": {"priorityClassName": "airflow-priority-class-name"}},
+            show_only=["templates/redis/redis-statefulset.yaml"],
+        )
+
+        assert "airflow-priority-class-name" == jmespath.search(
+            "spec.template.spec.priorityClassName",
+            docs[0],
+        )
 
 class TestRedisServiceAccount:
```

Comment:
Try to fix static checks

```suggestion
        )


class TestRedisServiceAccount:
```